Adjust the `install_wc` function to fix the PHP unit tests on GitHub Actions

eason9487 opened the pull request:
### Changes proposed in this Pull Request:

[WooCommerce repo is starting the move to be a monorepo](https://github.com/woocommerce/woocommerce/pull/30945). With the change, the [PHP unit tests on GitHub Actions](https://github.com/woocommerce/google-listings-and-ads/pull/1056/checks?check_run_id=3935120358) also broke down along with it.

![image](https://user-images.githubusercontent.com/17420811/137875990-ab899a89-7e5e-4191-930c-1b054eeb9499.png)

This PR aims to give a quick fix by adjusting the `install_wc` function in the **bin/install-wp-tests.sh** to fit with the new file structure of WooCommerce repo.

### Screenshots:

![image](https://user-images.githubusercontent.com/17420811/137876748-0c848e69-49a3-4130-a58b-ea7e8b5197bf.png)

### Detailed test instructions:

1. Go to [the Checks tab](https://github.com/woocommerce/google-listings-and-ads/runs/3936771130#step:7:37) for more details

### Changelog entry

eason9487 committed:
Adjust the install_wc function to fix the PHP unit tests on GitHub Actions.

Since the WooCommerce was changed to a monorepo, the `install_wc` function
in the install-wp-tests.sh needs adjustment to fit with the new file structure.

layoutd reviewed (approved):
Looks good and is working (for trunk, at least). 

Two notes: 
- I guess we'll have to hard-code the version where WC went monorepo in order to handle testing with older versions
- I wonder if the commands that use `$WC_DIR` and `$TMPDIR` should go in `"`" quotes in case they have spaces in them? I.e., `mkdir -p "$WC_DIR"`

eason9487 committed:
Add double quotes around all folder-related variables.

Address https://github.com/woocommerce/google-listings-and-ads/pull/1057#pullrequestreview-783369289

eason9487 commented:
> * I guess we'll have to hard-code the version where WC went monorepo in order to handle testing with older versions

Yes.

Maybe downloading the zip file from https://wordpress.org/plugins/woocommerce/ for the specified versions would avoid the hard-code.

> * I wonder if the commands that use `$WC_DIR` and `$TMPDIR` should go in `"`" quotes in case they have spaces in them? I.e., `mkdir -p "$WC_DIR"`

Thanks for reminding me! I added double quotes around all folder-related variables by https://github.com/woocommerce/google-listings-and-ads/pull/1057/commits/4e9468c54c52c7b1afe1f96a0e127416961c73df
